sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/KeyVaultAccessControlClientBuilder.java

```diff
@@ -330,6 +360,22 @@ public KeyVaultAccessControlClientBuilder retryPolicy(RetryPolicy retryPolicy) {
         return this;
     }
 
+    /**
+     * Sets the {@link RetryOptions} for all the requests made through the client.
+     * <p>
+     * Setting this is mutually exclusive with using {@link #retryPolicy(RetryPolicy)}.
+     *
+     * @param retryOptions The {@link RetryOptions} to use for all the requests made through the client.
+     *
+     * @return The updated {@link KeyVaultAccessControlClientBuilder} object.
+     */
+    @Override
+    public KeyVaultAccessControlClientBuilder retryOptions(RetryOptions retryOptions) {
+        Objects.requireNonNull(retryOptions, "'retryOptions' cannot be null.");
```

Comment:
Maybe users should be able to unset the retry options (make them `null`). If we don't want that, we should at least handle NPEs for such cases like this:
```suggestion
    if (retryOptions== null) {
        throw logger.logExceptionAsError(new NullPointerException("'retryOptions' cannot be null."));
    }
```
And include a `@throws` section in the method's Javadoc.